docs/trust-list.mdx

```diff
@@ -26,13 +26,44 @@ The [contentcredentials.org](https://contentcredentials.org/) site hosts the fol
 - **The temporary known anchor list** in https://contentcredentials.org/trust/anchors.pem contains the list of known anchor certificates. If an end-entity [certificate's chain](getting-started.mdx#signing-and-certificates) can be traced back to an anchor certificate on this list, the certificate is considered "known."
 - **The configuration file**, https://contentcredentials.org/trust/store.cfg, specifies the [Extended Key Usage (EKU)](https://datatracker.ietf.org/doc/html/rfc9336) values accepted for end-entity certificates. An end-entity certificate must have at least one of the EKUs in this list to be valid.
 
+## Checking your certificate
+
+Before requesting to add your signing certificate to the known certificate list, perform a preliminary "sanity check" on your certificate by following these steps:
+
+1. **Ensure that signing with the certificate doesn't have any validation errors** by using a C2PA Tool command like this:
+
+```
+c2patool ./image.jpg trust --allowed_list ./cert.pem
+```
+
+Confirm that the result does not contain a `validation_status` field, which indicates an error.
+
+1. **Confirm that the `signature_info.issuer` field in the manifest is correct**. This field determines what [Verify displays for the organization name](verify.mdx#title-and-signing-information) after "Issued by ...". In this example, replace "XYZ Inc." with the name of your organization:
+
+```
+c2patool ./image.jpg trust --allowed_list ./cert.pem \
+| jq --args '.manifests[].signature_info.issuer' "XYZ Inc."
```

Comment:
Is "XYZ Inc." supposed to be on a new line here as the response? I don't think providing it as another argument to `jq` does a comparison or anything.

Reply:
Right, removed.  I copied this from your original message and mistook it as being part of the command.

Reply:
Ah, understood